bundles/net.tourbook.common/src/net/tourbook/common/Messages.java

```diff
@@ -134,6 +134,8 @@
    public static String        Graph_Label_Power_Unit;
    public static String        Graph_Label_Speed;
    public static String        Graph_Label_Temperature;
+   //CUSTOM TRACKS
+   public static String        Graph_Label_Custom_Tracks;
```

Comment:
> Sometimes a comment is useful but I gave up to review and comment all occurences
> 
> ![mt custom tracks comments 2](https://user-images.githubusercontent.com/1283445/106463919-4592fe80-6498-11eb-8440-2a5cd9a77cfd.png)

I was using the comments in the beginning to easily find where I modified the MTB code, since I was and I'am still new to this code. So stop looking at the comments "//CUSTOM TRACKS" it is very easy to remove all of them at the very end. I can remove ALL of them now  if it is so disturbing for you.

on top of that in the Messages.java I put "Graph_Label_Custom_Tracks" on the wrong place, it should be put after  the last "Graph_Label_Cadence_Unit_RpmSpm" to respect de Alphabetical order. It has been corrected locally already

Reply:
I'm doing a review for a **final code** where it should not be modified afterwards to remove temporarily comments otherwise I would need to do **ANOTHER REVIEW** for the **final final code**. 

Until now I've spend many hours just for reviewing this feature and sometimes it is enough.